website/docs/components/code-block/partials/guidelines/overview.md

```diff
@@ -0,0 +1 @@
+The CodeBlock is used to display, format, and highlight the syntax of example code and snippets.
```

Comment:
```suggestion
The CodeBlock is used to display, format, and highlight the syntax of code snippets.
```

Reply:
Resolved in https://github.com/hashicorp/design-system/pull/1749/commits/19a092530be5f2419739f829a8d000a99a7cb4a7